### ISSUE TITLE
`PPOConfig.gamma` and `gae_lambda` are never forwarded, so GAE always runs at the base defaults

### ISSUE THREAD

**Issue by CodersAcademy006** [ellanorai/ThinkRL#98]
`PPOConfig` exposes `gamma` and `gae_lambda`, but `PPOAlgorithm.__init__` never forwards them. `thinkrl/algorithms/ppo.py:100-107`:

```python
super().__init__(
    policy_model=policy_model,
    ref_model=ref_model,
    optimizer=optimizer,
    learning_rate=config.learning_rate,
    kl_coeff=0.0,
    clip_grad_norm=config.clip_grad_norm,
    **kwargs,
)
```

`BaseRLHFAlgorithm.__init__` accepts `gamma`, `lambda_` and `normalize_advantages` at `algorithms/base.py:88-91` and defaults them to `0.99`, `0.95` and `True`. `compute_gae_advantages` reads `self.gamma` and `self.lambda_`, so GAE always runs at the base defaults no matter what the config says:

```
PPOConfig(gamma=0.5, gae_lambda=0.1)  ->  self.gamma=0.99, self.lambda_=0.95
```

Two things make this invisible. The defaults coincide numerically, so nothing looks wrong until someone tunes them, and the config field is named `gae_lambda` while the base parameter is named `lambda_`, so no amount of `**kwargs` forwarding would have bound it either.

`normalize_advantages` has the same gap. The config default and the base default are both `True`, so setting `PPOConfig(normalize_advantages=False)` still normalizes inside `compute_gae_advantages` and only skips the minibatch re-normalization at `:253`, which is half of what the flag says it does.

**Suggested fix**

Pass `gamma=config.gamma, lambda_=config.gae_lambda, normalize_advantages=config.normalize_advantages` in the `super().__init__` call, and consider renaming `gae_lambda` to `lambda_` for consistency with the base class.

I am happy to open a PR for this.
